pkgs/applications/terminal-emulators/kitty/default.nix

```
@@ -29,16 +29,21 @@
 with python3Packages;
 buildPythonApplication rec {
   pname = "kitty";
-  version = "0.28.1";
+  version = "0.29.0";
   format = "other";
 
   src = fetchFromGitHub {
     owner = "kovidgoyal";
     repo = "kitty";
     rev = "refs/tags/v${version}";
-    hash = "sha256-pAo+bT10rdQOf9j3imKWCCMFGm8KntUeTQUrEE1wYZc=";
+    hash = "sha256-FTitj43RFCNvSWInXHALyIljfcBBkaq/XI1ZA1k0glk=";
   };
-  vendorHash = "sha256-vq19exqsEtXhN20mgC5GCpYGm8s9AC6nlfCfG1lUiI8=";
+
+  goModules = (buildGoModule {
+    pname = "kitty-go-modules";
+    inherit src version;
+    vendorHash = "sha256-jk2EcYVuhV/UQfHAIfpnn8ZIZnwjA/o8YRXmpoC85Vc=";
+  }).go-modules;
```

```suggestion
  inherit = (buildGoModule {
    pname = "kitty-go-modules";
    inherit src version;
    vendorHash = "sha256-jk2EcYVuhV/UQfHAIfpnn8ZIZnwjA/o8YRXmpoC85Vc=";
  }) go-modules;
```
naive question: why don't we do this and remove go-modules from passthru and rename the bash variable below?

---

Are you suggesting to effectively revert https://github.com/NixOS/nixpkgs/pull/241707 ?